docs/Banners.md

```diff
@@ -0,0 +1,45 @@
+# Banners
+
+The HCMI Portal UI supports displaying informational banners at the top of each page that can be configured through environment variables.
+
+![Banners example](./images/banners.png "Banners example")
+
+## Usage
+
+To configure banners, provide an array of JSON objects representing the contents of each banner in the `REACT_APP_BANNERS` environment variable for the UI project (`ui/.env`). Multiple banners can be present at once.
+
+### Banner JSON
+
+Each object should be structured as follows:
+```json
+{
+  "title": string,
+  "message": string,
+  "type": INFO | WARNING | CRITICAL,
+  "dismissible": boolean,
+}
+```
+
+The object properties are as follows:
+* `title` <sup>*optional*</sup>
+  * Banner title, supports Markdown
+  * Example: `"HCMI Portal Publication [Now Available](https://example.com/)"`
+* `message` <sup>*optional*</sup>
+  * Banner message, supports Markdown
+  * Example: `"We are proud to announce a new publication about the impact of HCMI. Read more here: [https://example.com/](https://example.com/)"`
+* `type` <sup>*optional*</sup>
+  * Banner type
+  * Values: `INFO` | `WARNING` | `CRITICAL`
+  * Default: `INFO`
+* `dismissible` <sup>*optional*</sup>
+  * Boolean representing whether the banner should be dismissible or not
```

Comment:
@mistryrn
```suggestion
  * Tells the banner whether users should able to "close" the banner (e.g. display an `x` on the top right corner).
```

Note to self + @MitchellShiell: Banners should also be able to close temporarily and reopen later (i.e. "snooze").